swift-reactive-blerpc/src/main/java/com/blerpc/ios/MessageGenerator.java

```diff
@@ -0,0 +1,153 @@
+                fieldContext.protoType = field.getTypeName().replace(protoFile.getPackage(), "")
+                        .replace(".", "");
+
+                if (fieldContext.type.equals("TYPE_ENUM")) {
+                    fieldContext.isEnum = true;
+                    fieldContext.isProtoObject = false;
+                    fieldContext.isPrimitiveType = false;
+                } else if (fieldContext.type.equals("TYPE_MESSAGE")) {
+                    fieldContext.isEnum = false;
+                    fieldContext.isProtoObject = true;
+                    fieldContext.isPrimitiveType = false;
+                } else {
+                    fieldContext.isEnum = false;
+                    fieldContext.isProtoObject = false;
+                    fieldContext.isPrimitiveType = true;
+                }
+
+                switch (fieldContext.type) {
+                    case "TYPE_INT32":
+                        fieldContext.swiftType = "ProtoType.int32";
+                        break;
+                    case  "TYPE_BYTES":
+                        fieldContext.swiftType = "ProtoType.byte";
+                        break;
+                    case  "TYPE_BOOL":
+                        fieldContext.swiftType = "ProtoType.bool";
+                        break;
+                    default:
+                        fieldContext.swiftType = "ProtoType.unknown";
+                        break;
+                }
+
+                fieldContext.toByte = field.getOptions().getExtension(Blerpc.field).getToByte();
+                fieldContext.fromByte = field.getOptions().getExtension(Blerpc.field).getFromByte();
+                arrayFields.add(fieldContext);
+            }
+
+            messageContext.fields = ImmutableList.copyOf(arrayFields);
+            arrayMessages.add(messageContext);
+        }
+
+        return arrayMessages.stream();
+    }
+
+    private Stream<AbstractMap.SimpleEntry<DescriptorProtos.FileDescriptorProto, DescriptorProtos.SourceCodeInfo.Location>> getFileLocations(
+            DescriptorProtos.FileDescriptorProto file) {
+        return file.getSourceCodeInfo()
+                .getLocationList()
+                .stream()
+                .map(location -> new AbstractMap.SimpleEntry<>(file, location));
+    }
+
+    private boolean hasPackage(DescriptorProtos.FileDescriptorProto file) {
+        return !file.getPackage().isEmpty();
+    }
+
+    private String extractPackageName(DescriptorProtos.FileDescriptorProto proto) {
+        String javaPackage = proto.getOptions().getJavaPackage();
+        return !javaPackage.isEmpty() ? javaPackage : proto.getPackage();
+    }
+
+    private String extractSwiftPackageName(DescriptorProtos.FileDescriptorProto proto) {
+        String packageName = proto.getPackage();
+        String[] splittedPackageName = packageName.split("\\.");
+        StringBuilder swiftPackageName = new StringBuilder();
+
+        for (String substring : splittedPackageName) {
```

Comment:
`return splittedPackageName.stream().map(this::upperCaseFirstLetter).collect(Collectors.joining( "_" ));`